runtime/src/bank/tests.rs

```diff
@@ -1951,6 +1951,7 @@ fn test_load_and_execute_commit_transactions_fees_only() {
                 loaded_accounts_count: 2,
                 loaded_accounts_data_size: nonce_size as u32,
             },
+            fee_payer_post_balance: genesis_config.rent.minimum_balance(0) - 1 - 5000,
```

Comment:
Somehow we don't seem to have tests for successful and failed executed transactions for `load_execute_and_commit_transactions`, can you add those and check that the fee payer post balance is correct?

Reply:
c66056b2c2 added tests for success and failed execution cases.